integration/airflow/tests/integration/airflow/dags/bigquery_orders_popular_day_of_week.py

```diff
@@ -0,0 +1,107 @@
+
+import time
+
+from airflow.operators.python_operator import PythonOperator
+
+from openlineage.airflow import DAG
+from airflow.contrib.operators.bigquery_operator import BigQueryOperator
+from airflow.utils.dates import days_ago
+
+from openlineage.client import set_producer
+set_producer("https://github.com/OpenLineage/OpenLineage/tree/0.0.1/integration/airflow")
+
+default_args = {
+    'owner': 'datascience',
+    'depends_on_past': False,
+    'start_date': days_ago(7),
+    'email_on_failure': False,
+    'email_on_retry': False,
+    'email': ['datascience@example.com']
+}
+
+dag = DAG(
+    'bigquery_orders_popular_day_of_week',
+    schedule_interval='@once',
+    default_args=default_args,
+    description='Determines the popular day of week orders are placed.'
+)
+
+PROJECT_ID = 'speedy-vim-308516'
+DATASET_ID = 'airflow_integration'
+CONNECTION = 'bq_conn'
```

Comment:
Let's open some issues to get some of these integration tests to only run on merge to main and also to get a dedicated GCP project for testing- just so we avoid the potential of running up somebody's credit card bill 😉